packages/common/src/classes/testTypes/Constants.ts

```diff
@@ -154,3 +154,69 @@ export const ANNUAL_WITH_CERTIFICATE: ITestTypeList = {
     '199',
   ],
 };
+
+export const CENTRAL_DOCS_TEST_TYPES = {
```

Comment:
Should we call this `CENTRAL_DOCS_TEST` to keep it consistent? `IVA_TEST`, `LEC_TEST` etc